src/core/data.tf

```diff
@@ -184,3 +184,18 @@ data "azurerm_storage_account" "cdnassets" {
   name                = "iopstcdnassets"
   resource_group_name = var.common_rg
 }
+
+data "azurerm_eventhub" "payment_updater_evh" {
```

Comment:
@pasqualedevita we referenced an event hub defined somewhere else in this way. Is it the right way to do it?

Reply:
yes it's ok :)